src/signer/src/main/java/ee/ria/xroad/signer/certmanager/OcspClientWorker.java

```diff
@@ -454,37 +451,39 @@ boolean isCachedOcspResponse(String certHash) throws Exception {
 
             return chain.getAllCertsWithoutTrustedRoot();
         } catch (Exception e) {
-            log.error("Error getting certificate chain for certificate " + cert.getSubjectX500Principal(), e);
+            log.error("Error getting certificate chain for certificate {}", cert.getSubjectX500Principal(), e);
         }
 
         return emptyList();
     }
 
     private void initializeDiagnostics() {
-        for (X509Certificate caCertificate : GlobalConf.getAllCaCerts()) {
+
```

Comment:
It's sometimes a bit difficult (or slow) to study the code and understand, what happens and why. 

Maybe some inline comments, or short javadocs even for private methods, would help (e.g. `initializeDiagnostics` could have a short description that outlines it's purpose)